### ISSUE TITLE
Cypress requests and cancel image files over and over again since 4.12

### ISSUE THREAD

**Issue by aldarund** [cypress-io/cypress#9041]
### Current behavior
Cypress do loop requests to images and than cancel them and than do it again and again and again and its hog up resources
Here is video how it behave https://www.dropbox.com/s/t34a47ro43ww73x/2020-10-30_14-43-50.mp4?dl=0
Here is screenshot https://i.imgur.com/QTkjWqn.png
It basically makes cypress unusable as it hogs on cpu and memory when it happens and works really slow

### Desired behavior
No looping cancelled requests to images

### Test code to reproduce
I don't have code since its private project and it happens when running locally on webpack project with HMR enabled. When running against prod site it still happens but way less severe.
Here is a logs from cypress with this behaviour
https://www.dropbox.com/s/2gvse0el7oidmfb/logs.txt?dl=0
Test itself doesn't matter, it happens on every test..

### Versions

cypress 4.11 and below - works fine without this issue. cypress 4.12 and all versions ( well didn't tested all but 4.12.1, 5.0,5.1 and 5.5) forward to 5.5 - have this issue
I checked changelog for 4.12 and nothing clicked that could cause this

**Comment by philiiiiiipp** [comment]
We are experiencing the same issue. I noticed that it does not happen with every image, one image where I did see it happening is using the place image api `http://placeimg.com/640/480`, using another random image from the internet `https://helpx.adobe.com/content/dam/help/en/stock/how-to/visual-reverse-image-search/jcr_content/main-pars/image/visual-reverse-image-search-v2_intro.jpg` and it seems to be working fine.

**Comment by philiiiiiipp** [comment]
Checking a little further, I believe that this behavior stops once we use https instead of http for the image.

`http://placeimg.com/640/480` <- Gets re-requested every time
`https://placeimg.com/640/480` <- Only once

**Comment by jennifer-shehane** [comment]
Thanks for opening an issue. 

This is a duplicate of https://github.com/cypress-io/cypress/issues/8679 which has a PR open to be fixed https://github.com/cypress-io/cypress/pull/9050
